### ISSUE TITLE
Update AddTag support multiple tags

### ISSUE THREAD

**Issue by matthewvalimaki** [mailgun/mailgun-go#96]
I believe this would make the function a bit more flexible.
```
func (m *Message) AddTag(tag ...string) {
	m.tags = append(m.tags, tag...)
}
```
While we're at it we could do a check for the maximum number of tags (currently stated in the docs to be 3) and throw an error.
```
func (m *Message) AddTag(tag ...string) error {
	if len(m.tags) > 2 {
		return errors.New("Message tag limit (3) reached.")
	}
	m.tags = append(m.tags, tag...)
	return nil
}
```

**Comment by thrawn01** [comment]
I like this! PR?

**Comment by thrawn01** [comment]
#102

**Comment by thrawn01** [comment]
Released under v3.0.0 tag
